.github/workflows/lint.yml

```diff
@@ -15,14 +15,17 @@ jobs:
   golangci:
     strategy:
       matrix:
-        module: ["chain", "cosmosutil", "node", "types", "util", "wallet"]
+        module: ["cosmos", "general"]
```

Comment:
```suggestion
        module: ["cosmos", "core"]
```

I would rename the `general` package to `core`.  I think that is a bit clearer and standard

Reply:
i like that